src/api/ipc-channels.ts

```diff
@@ -116,6 +115,7 @@ export enum IpcChannels {
   DEPENDENCY_ACCEPT_ALL = 'DEPENDENCY_ACCEPT_ALL',
   DEPENDENCY_REJECT = 'DEPENDENCY_REJECT',
   DEPENDENCY_REJECT_ALL = 'DEPENDENCY_REJECT_ALL',
+  DEPENDENCY_SUMMARY = 'DEPENDENCY_SUMMARY',
```

Comment:
Could this have a better name? Like DEPENDENCY_MANIFEST_FILE_SUMMARY ?

The channel DEPENDENCY_GET_ALL returns a list of the type Dependency, which is an individual dependency. But here is not the same, we are looking for the dependency manifest files.